modules/ROOT/pages/afm-create-monitor.adoc

```diff
@@ -7,34 +7,45 @@ This procedure describes creating monitors via the Functional Monitoring UI.
 [[before-you-begin]]
 == Before You Begin
 
-* Ensure that you have the following permissions set in Access Management:
-+
-** *Anypoint Monitoring > Viewer* or *Anypoint Monitoring > Administrator* 
+Ensure that you have the following permissions set in Access Management:
+
+* *Anypoint Monitoring > Viewer* or *Anypoint Monitoring > Administrator* 
 +
 and
 +
-** *Exchange > Exchange Administrator* or *Exchange > Exchange Contributor*
-+
+* *Exchange > Exchange Administrator* or *Exchange > Exchange Contributor*
+
 An Anypoint Platform administrator for your organization grants permission. See xref:access-management::managing-permissions.adoc[Manage Permissions].
 
-* Determine which endpoints to test and by which REST methods to test them.
-* Determine the expected return status codes.
-* Determine the test interval.
-+
-The shortest interval is 15 minutes. Monitors can also run hourly, daily, weekly, or monthly. The schedule begins immediately after creating the monitor, and you can modify the schedule after the monitor has been created. For example:
+[[create-a-monitor]]
+== Create a Monitor
 
-** Set an hourly schedule to start at a certain number of minutes after the hour begins.
-** Set a daily, weekly, or monthly schedule to start at a certain time of day.
+To create a monitor:
 
-* Determine the location in which to execute the monitor.
+. Click *Create Monitor* in the *Functional Monitoring* section of Anypoint Monitoring.
+. Under *Step 1 - Setup the monitor*:
+.. Specify a monitor name.
+.. Select a testing location. 
+.. Set the monitoring schedule. 
+. Click the *Step 2 - Select endpoints* tab and enter the following for each endpoint: 
+.. By default, the monitor validates self-signed certifications for each endpoint, so *Validate SSL Certificate* is toggled on. To allow unsafe self-signed certifications for an endpoint, toggle *Validate SSL Certificate* off.
```

Comment:
I suggest flipping the sentences to make this bullet more parallel with the others:

Optionally, allow unsafe self-signed certifications for an endpoint, by toggling *Validate SSL Certificate* off. (Validate SSL Certificate* is toggled on by default.)

Reply:
It is really recommended that they leave it toggled on. I followed the original wording for the most part when they recently changed the level of the toggle from the monitor level to the endpoint level. I'll leave this one as-is for now since it was also approved as it was before by Eng.